content/cloud/noops-sql-db.md

```diff
@@ -1,29 +1,30 @@
 title = "NoOps SQLite Database"
 template = "cloud_main"
 date = "2023-07-26T16:00:00Z"
-enable_shortcodes = false
+enable_shortcodes = true
 [extra]
 url = "https://github.com/fermyon/developer/blob/main/content/cloud/noops-sql-db.md"
 
 ---
 
 - [Accessing Private Beta](#accessing-private-beta)
 - [Quotas And Service Limitations For the Database in Fermyon Cloud](#quotas-and-service-limitations-for-the-database-in-fermyon-cloud)
-- [How To:](#how-to)
-	- [Prepare a Database](#prepare-a-database)
-	- [Manage the Database Lifecycle](#manage-the-database-lifecycle)
+- [How To](#how-to)
+  - [Database Access From Many Components](#database-access-from-many-components)
```

Comment:
The grammar of the "how to" subtitles doesn't flow from the top-level title - "How to database access", "How to deleting", etc.

Reply:
Thanks @itowlson 
I am updating this in the next push to align the language for each heading.